vars/provisionNodes.groovy

```diff
@@ -220,6 +256,8 @@ EOF'''
       if (inst_rpms) {
          provision_script += '''\nyum -y erase ''' + inst_rpms
       }
+      provision_script += '\nrpm --import https://download.fedoraproject.org/' +
+                                         'pub/epel//RPM-GPG-KEY-EPEL-7'
```

Comment:
But this is not necessary on vanilla installations.  All that is necessary is to do on any EL7 machine is:

```
# yum install epel-release
```

and EPEL is ready to go.  That's why this has worked for the many months that it has without having to do this key import.

So the question is why is it not working now?  Can you paste (a link, if you prefer, to) the error you get without this explicit key addition?  My concern is that something else has subtly broken which we'd be better to uncover now rather than later.

Reply:
It appears to be a side effect of using yum add-repo with a Nexus group repository.
Installs fail on validating the repository until any the rpm --import /etc/pki/rpm-gpg/* is run.
If any additional GPG keys not already installed on the system are needed, then they also need to be imported with the rpm command.

I will be adjusting this PR to also verify that the ipmctl libraries install as they need their own GPG key.